client-java/controller/pom.xml

```diff
@@ -144,6 +144,11 @@
             <groupId>org.testcontainers</groupId>
             <artifactId>testcontainers</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.hamcrest</groupId>
+            <artifactId>hamcrest-all</artifactId>
+            <scope>compile</scope>
+        </dependency>
```

Comment:
this should be removed. hamcrest-all is already imported in that pom, with scope <test> (declared in the parent-pom). See discussion on adding libraries to controller in comments at the top of this file

Reply:
For the record. Updated the scope of hamcrest for the controller to "provided".